modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java

```diff
@@ -929,6 +929,18 @@ private boolean _isPermanentRedirect(long companyId) {
 		return false;
 	}
 
+	private boolean _isShowRedirectAlternativeFriendlyURLMessage(
```

Comment:
Why not call this `alternativeLayoutFriendlyURLMessage` for consistency with the attribute name that was used originally? Use that also for other methods/lang keys.

Reply:
Done